reporter.go

```diff
@@ -234,7 +235,7 @@ func (r *remoteReporter) processQueue() {
 				atomic.AddInt64(&r.queueLength, -1)
 				if flushed, err := r.sender.Append(span); err != nil {
 					r.metrics.ReporterFailure.Inc(int64(flushed))
-					r.logger.Error(err.Error())
+					r.logger.Error(fmt.Sprintf("span %q: %s", span.OperationName(), err.Error()))
```

Comment:
`s/span %q/error reporting span '%q'/`

Reply:
Hi Yuri, `%q` will be automatically surrounded by quotes.